OpenTrons/opentrons-api
# docs(api): refactor-tip-handling

# Overview

This PR is for managing changes made to the "Manipulating Pipette Tips" section of the doc, [Building Block Commands](https://docs.opentrons.com/v2/new_atomic_commands.html). Following the plan laid out in [this Figma file](https://www.figma.com/file/WsKj7gklIgDx68DkzegdaN/Commands-PAPI-docs-merge-strategies?type=whiteboard&node-id=0%3A1&t=souDTlzeZ3A0tmOj-1). This PR is for the first, small branch section in Merge Strategy 1.

The manipulating tips section has 4 parts:

- Picking Up a Tip
- Dropping a Tip
- Returning a Tip
- Iterating Through Tips

# Test Plan

-  Check code in the Black formatter and use that style.
- Make sure links work.
- The usual grammar, style, clarity stuff.

# Changelog

This refactoring introduces significant changes to the text used in the manipulating tips section. This includes things like:

- new code samples
- new and revised text
- some shortcuts/anchors
- fix section headers to use proper style

# Review requests

The usual suspects.

# Risk assessment

Low risk. Docs changes only.
